lib/src/main/java/com/smileidentity/models/Bvn.kt

```diff
@@ -0,0 +1,109 @@
+package com.smileidentity.models
+
+import com.smileidentity.SmileID
+import com.squareup.moshi.Json
+import com.squareup.moshi.JsonClass
+
+@JsonClass(generateAdapter = true)
+data class BvnToptRequest(
+    @Json(name = "country")
+    val country: String,
+    @Json(name = "id_number")
+    val idNumber: String,
+    @Json(name = "id_type")
+    val idType: String,
+    @Json(name = "partner_id")
+    val partnerId: String = SmileID.config.partnerId,
+    @Json(name = "signature")
+    val signature: String,
+    @Json(name = "timestamp")
+    val timestamp: String,
```

Comment:
For requests, we should use this definition in order to continue supporting API key based approach, if the partner is using that

```suggestion
    @Json(name = "timestamp") val timestamp: String = System.currentTimeMillis().toString(),
    @Json(name = "signature") val signature: String = calculateSignature(timestamp),

```

Reply:
same thing on the other `Request` classes